### DISCUSSION TITLE
NET (Malta)

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#804] [Q&A]
### Channel Name

NET

### Channel ID

NetTV.mt

### Website

https://netondemand.mt/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by ARAB-IPTV** [comment]
#EXTINF:-1 tvg-ID="" tvg-name="NET" tvg-logo="http://frontend.smashmalta.com/iconsnew/net.png" group-title="",NET
http://a5.smashmalta.com:82/live/isuku/midoriya/4.ts

**Comment by R4IN2007** [comment]
https://netondemand.mt:8443/live/net/chunklist_w2097109062_DVR_tkdG9rZW5zdGFydHRpbWU9MTY1ODk2NDkxNSZ0b2tlbmVuZHRpbWU9MTY1OTAwODExNSZ0b2tlbmhhc2g9NlJhOEZ5UDJESWg1OW5hX3M5TEJUclR4NTR0LWlsWGVoWWlEZ0pERDV0OD0=.m3u8

**Comment by AntiPontifex** [comment]
Dead link: please close.

**Comment by freearhey** [comment]
http://a5.smashmalta.com:82/live/isuku/midoriya/4.ts (offline)
<img width="684" alt="image" src="https://user-images.githubusercontent.com/7253922/232229970-68bcaac3-de85-4751-9d2d-922e0ea32456.png">

https://netondemand.mt:8443/live/net/chunklist_w2097109062_DVR_tkdG9rZW5zdGFydHRpbWU9MTY1ODk2NDkxNSZ0b2tlbmVuZHRpbWU9MTY1OTAwODExNSZ0b2tlbmhhc2g9NlJhOEZ5UDJESWg1OW5hX3M5TEJUclR4NTR0LWlsWGVoWWlEZ0pERDV0OD0=.m3u8 (offline)
<img width="833" alt="image" src="https://user-images.githubusercontent.com/7253922/232230209-c6bfdeee-8627-43fb-8ac2-5c619890a999.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/mt_smash_plus.m3u#L12-L13
